dmutils/documents.py

```diff
@@ -132,56 +134,93 @@ def upload_service_documents(uploader, upload_type, documents_url, service, requ
     return files, errors
 
 
-# some of the more... seemingly oddly specific functions here are apparently referenced by validators, hence their
-# existence
-
-
 def file_is_not_empty(file_contents):
     return not file_is_empty(file_contents)
 
 
-def file_is_empty(file_contents):
-    empty = len(file_contents.read(1)) == 0
-    file_contents.seek(0)
-    return empty
+def file_is_empty(file_object):
+    return len(get_first_128_bytes(file_object)) == 0
 
 
 def file_is_less_than_5mb(file_):
-    return get_file_size(file_) < FILE_SIZE_LIMIT
+    return get_file_size(file_) < 5400000
+
+
+def get_first_128_bytes(file_object):
+    """Get the contents of a file like object but replace the pointer when we're finished"""
+    position = file_object.tell()
+    file_object.seek(0)
+    contents = file_object.read()
```

Comment:
Should this be `file_object.read(128)`?

Reply:
Doh! Good catch, yeah needs to be 128. Could also be `read()[:127]` but that loads the whole file I think